docs/change-log/_index.md

```diff
@@ -0,0 +1,339 @@
+(change-log)=
+# Change log
+- [v1.2.1](#v1-2-1)
+- [v1.2.0](#v1-2-0)
+- [v1.1.3](#1-1-3)
+- [v1.0.6](#v1-0-6)
+- [v1.0.5](#v1-0-5)
+- [v1.0.4](#v1-0-4)
+- [v1.0.3](#v1-0-3)
+- [v1.0.2](#v1-0-2)
+- [v1.0.0](#v1-0-0)
+- [Open issues](#open-issues)
+- [Limitations](#limitations)
+- [Deprecations](#deprecations)
+
+## v1.2.1
+
+### New and updated features
+
+#### Feature store
+- Supports ingesting Avro-encoded Kafka records. [View in Git](https://github.com/mlrun/mlrun/issues/2649).
+
+### Closed issues
+- When running a workflow with `local=True`, the serving function is updated. [View in Git](https://github.com/mlrun/mlrun/issues/2716).
```

Comment:
I don't understand what it fixes

Reply:
Pls see https://jira.iguazeng.com/browse/ML-3141 - and add a description. Tnx